packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/inspections/quickfixes/OpenConnectionChooserQuickFix.kt

```diff
@@ -0,0 +1,29 @@
+package com.mongodb.jbplugin.inspections.quickfixes
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.fileEditor.FileEditorManager
+import com.intellij.openapi.project.Project
+import com.mongodb.jbplugin.editor.MdbJavaEditorToolbar
+import kotlinx.coroutines.CoroutineScope
+
+/**
+ * This quickfix opens a modal with the connection chooser.
+ *
+ * @param message
+ * @param coroutineScope
+ */
+class OpenConnectionChooserQuickFix(
+    private val coroutineScope: CoroutineScope,
+    private val message: String,
+) : LocalQuickFix {
+    override fun getFamilyName(): String = message
+
+    override fun applyFix(
+        project: Project,
+        descriptor: ProblemDescriptor,
+    ) {
+        val editor = FileEditorManager.getInstance(project).selectedTextEditorWithRemotes[0] ?: return
```

Comment:
Out of curiosity what is the difference between selectedTextEditorWithRemotes and selectedTextEditor. Asking because I am using selectedTextEditor in my work to get a reference to current text editor.

Reply:
selectedTextEditorWithRemotes supports editors that are created by "Code with me" (IntelliJ pairing mechanism for remote working) and selectedTextEditor works only with local editors (no Code with me support)

Reply:
I see then I will switch my implementation as well to use the same, thank you for the context <3